packages/skeleton-react/src/lib/components/Accordion/Accordion.tsx

```diff
@@ -1,39 +1,29 @@
 'use client';
 
-import { motion, AnimatePresence } from 'framer-motion';
-import React, { createContext, useContext, useEffect, useState } from 'react';
+import { createContext, useContext, useId } from 'react';
+import type { FC } from 'react';
+import * as accordion from '@zag-js/accordion';
+import { useMachine, normalizeProps } from '@zag-js/react';
 
-import {
-	AccordionContextState,
-	AccordionControlProps,
-	AccordionItemContextState,
-	AccordionItemProps,
-	AccordionPanelProps,
-	AccordionProps
-} from './types.js';
+import type { AccordionContextState, AccordionControlProps, AccordionItemProps, AccordionPanelProps, AccordionProps } from './types.js';
+import { noop } from '../../internal/noop.js';
 
 // Contexts ---
 
 export const AccordionContext = createContext<AccordionContextState>({
 	animDuration: 0.2,
 	iconOpen: '-',
```

Comment:
iconOpen and iconClose are not options in the Svelte API, is this intentional?

Reply:
They are options, they're just implemented with different tech per framework:

1. In Svelte we use snippets
2. In React we use React.Node (props that accept template data)

Both are accounted for and use the same `iconX` naming scheme in the data and list of props.

You may have missed it for Svelte because the props are implemented on the parent, the passed down to the items via the Context API. That way you only have to set them once.